cmd/capibmadm/cmd/powervs/port/create.go

```diff
@@ -0,0 +1,136 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package port
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	client "github.com/IBM-Cloud/power-go-client/clients/instance"
+	"github.com/IBM-Cloud/power-go-client/power/models"
+
+	"sigs.k8s.io/cluster-api/cmd/clusterctl/log"
+
+	"sigs.k8s.io/cluster-api-provider-ibmcloud/cmd/capibmadm/clients/iam"
+	"sigs.k8s.io/cluster-api-provider-ibmcloud/cmd/capibmadm/clients/powervs"
+	"sigs.k8s.io/cluster-api-provider-ibmcloud/cmd/capibmadm/options"
+	"sigs.k8s.io/cluster-api-provider-ibmcloud/cmd/capibmadm/printer"
+	"sigs.k8s.io/cluster-api-provider-ibmcloud/cmd/capibmadm/utils"
+)
+
+type portCreateOptions struct {
+	networkID   string
+	ipAddress   string
+	description string
+}
+
+// CreateCommand create powervs network port.
+func CreateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Create PowerVS Port",
+		Example: `
+# Create PowerVS port, provided networkID and IPAddress
+export IBMCLOUD_API_KEY=<api-key>
+capibmadm powervs port create --network <netword-id/network-name> --ip-address <IP-address> --description <description> --service-instance-id <service-instance-id> --zone <zone>`,
```

Comment:
lets avoid optional field from example command
```suggestion
capibmadm powervs port create --network <netword-id/network-name> --description <description> --service-instance-id <service-instance-id> --zone <zone>`,
```